content/post/go-git-v4.md

```diff
@@ -0,0 +1,73 @@
+---
+author: campoy
+date: 2017-12-07
+title: "Announcing the latest go-git!"
+image: /post/go-git-v4/go-git.png
+description: "After a year of intense work, we're happy to announce the latest and best release of go-git ever. go-git v4 includes many new features, making it the most used and feature complete git library written in Go, and in use on production at companies like source{d} and keybase."
+categories: ["git", "announcement", "OSS", "technical"]
+---
+
+After a bit over a year of intense work, we're happy to announce the
+latest and best release of [go-git](https://github.com/src-d/go-git) yet!
+`go-git v4` includes many new features, making it the most used and feature
+complete git library written in Go.
+
+## what's new?
+
+A year and almost 800 commits have passed since `go-git v3.2.0` was released,
+our amazing contributors have really been very busy. But all of this effort
+was worth it and the results definitely show.
+
+Many (really [many](https://github.com/src-d/go-git/releases/tag/v4.0.0-rc1))
+new features have been added to `go-git`, but our favorites are:
+
+- TODO
```

Comment:
leftover

Reply:
the list of features is huge, you can check the list changes from the v4.rc14 to the stable version:
https://gist.github.com/mcuadros/61c8fb7bbb384f4877410b24e80156f8

So I thing is better include a paragrapah saying something like: 
```
After almost 18 months of development, more than +700 commits, and 15 release candidates, we are pleasant to release the v4, covering almost all the common used git functionalities.
```

That I used on the release notes.